mongo/integration/collection_test.go

```diff
@@ -74,6 +74,48 @@ func TestCollection(t *testing.T) {
 			assert.True(mt, ok, "expected error type %v, got %v", mongo.WriteException{}, err)
 			assert.NotNil(mt, we.WriteConcernError, "expected write concern error, got %+v", we)
 		})
+
+		// Require 3.2 servers for bypassDocumentValidation support.
```

Comment:
There should be another `mt.Run`/`mt.RunOpts` call here so all of the table tests are run inside a larger context. Right now, the tests would be named something like `TestCollection/insert_one/only_nil_is_passed`. If we add another level of nesting, it'd be `TestCollection/insert_one/options_are_converted/only_nil_is_passed`.

Reply:
Done.